src/frontend/planner_test/tests/testdata/input/explain_dot_format.yaml

```diff
@@ -40,5 +40,17 @@
     COALESCE((SELECT b FROM t2 WHERE t1.a = t2.c), 0) col17,
     COALESCE((SELECT b FROM t2 WHERE t1.a = t2.c), 0) col18
     from t1;
+  expected_outputs:
+    - explain_output
+- name: test dot output format (distsql, batch)
+  sql: |
+    CREATE TABLE t (v1 int);
+    explain (distsql, format dot) SELECT approx_percentile(0.5) WITHIN GROUP (order by v1) from t;
```

Comment:
somehow this will fail in test, but it works when i type the query into a rw instance. cc @kwannoel

Reply:
Error: Scheduler error: Streaming vnode mapping not found for fragment 4294967294